packages/ui/src/components/Markdown/index.tsx

```diff
@@ -10,19 +10,28 @@ const StyledMarkdown = styled.div`
   a {
     color: ${({theme}) => theme.colors.textSecondary};
   }
-  .mdx a {
+
+  .markdown a {
     padding: 0 ${paddings.small};
   }
 `;
 
-const Markdown = ({children}: {children: React.ReactNode}) => {
-  const mdxComponents = {code: Highlighter};
-
-  return (
-    <MDXProvider components={mdxComponents}>
-      <StyledMarkdown>{children}</StyledMarkdown>
-    </MDXProvider>
+const Markdown = ({source}: {source: string}) => {
+  const elements = ReactHtmlParser(source);
+  const highlighted = elements.map((el: React.ReactElement, idx: number) =>
```

Comment:
Is "index" so hard to type? 🙃

Reply:
Sure, do you want me to use element instead of el as well for consistency?

Reply:
Good shout.